### ISSUE TITLE
RaspberryPi, python 3.9.2 and asyncio-mqtt-0.14.0 getting ImportError on 'Concatenate' from 'typing_extensions'

### ISSUE THREAD

**Issue by asedagha** [sbtinstruments/asyncio-mqtt#154]
Hi,

I am testing  `asyncio-mqtt` on a raspberrypi but that should not really matter, but I am getting the following error:

 ```
File "./main.py", line 23, in <module>
    import asyncio_mqtt as aiomqtt
  File "/.local/lib/python3.9/site-packages/asyncio_mqtt/__init__.py", line 3, in <module>
    from .client import Client, ProtocolVersion, ProxySettings, TLSParameters, Will
  File ".local/lib/python3.9/site-packages/asyncio_mqtt/client.py", line 33, in <module>
    from typing_extensions import Concatenate, ParamSpec
ImportError: cannot import name 'Concatenate' from 'typing_extensions' (/usr/lib/python3/dist-packages/typing_extensions.py)
```

Since I am on  `Python 3.9.2 `,  `typing_extensions` is already installed so I am a bit confused.

**Comment by asedagha** [comment]
Not sure why but I had to update type-extensions

`pip install -Iv typing-extensions`

**Comment by JonathanPlasse** [comment]
Thanks for reporting.
It has been fixed in 2575d21.
